### PR TITLE
Reformat: Fix pep8 and yaml linting issues

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Expand test coverage of `make linting`:
  - lint-python -> add coverage for all plugins
  - lint-yaml  -> include molecule scenarios inventory

Address all errors and warnings raised by pep8 and yamllint from: `make linting`

## How to verify

Run `make linting` from the root of repo and no more "errors" reported:

```shell
(.venv) ➜  ansible-avd git:(fix-linting-issues) make linting
sh .github/lint-yaml
------------------------------------------------------------------------------------------------------------
Validating YAML files in AVD collection ...
------------------------------------------------------------------------------------------------------------
ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/aaa.yml
  110:301   warning  line too long (594 > 300 characters)  (line-length)

ansible_collections/arista/avd/molecule/upgrade_eos_cli_config_gen/inventory/host_vars/aaa.yml
  95:301    warning  line too long (594 > 300 characters)  (line-length)

------------------------------------------------------------------------------------------------------------
sh .github/lint-python
Validating PEP8 compliance on Plugins...
------------------------------------------------------------------------------------------------------------
  - plugins/actions validation

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

------------------------------------------------------------------------------------------------------------
  - plugins/filter validation

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

------------------------------------------------------------------------------------------------------------
  - plugins/module_utils validation

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

------------------------------------------------------------------------------------------------------------
  - plugins/modules validation
************* Module add_toc
ansible_collections/arista/avd/plugins/modules/add_toc.py:101: [R1732(consider-using-with), main] Consider using 'with' for resource-allocating operations
ansible_collections/arista/avd/plugins/modules/add_toc.py:108: [R1732(consider-using-with), main] Consider using 'with' for resource-allocating operations
************* Module inventory_to_container
ansible_collections/arista/avd/plugins/modules/inventory_to_container.py:1: [R0801(duplicate-code), ] Similar lines in 2 files
==add_toc:[20:29]
==configlet_build_config:[20:29]
from __future__ import (absolute_import, division, print_function)
__metaclass__ = type

ANSIBLE_METADATA = {'metadata_version': '1.0.0',
                    'status': ['preview'],
                    'supported_by': 'community'}

DOCUMENTATION = r'''
---

------------------------------------------------------------------
Your code has been rated at 9.86/10 (previous run: 9.91/10, -0.05)

------------------------------------------------------------------------------------------------------------
  - plugins/test validation

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

------------------------------------------------------------------------------------------------------------
```

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/plugins/modules/inventory_to_container.py [hunk 1]
```diff
@@ -316,7 +316,7 @@ def get_devices(dict_inventory, search_container=None, devices=None, device_filt
     search_container : str, optional
         Container to look for, by default None
     devices : str, optional
-        List of found devices attached to container, by default list()
+        List of found devices attached to container, by default list []
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/plugins/modules/inventory_to_container.py]: should be ", by default empty list []" or ", by default []"

File: ansible_collections/arista/avd/plugins/module_utils/strip_empties.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def strip_null_from_data(data):
     return data
 
 
-def strip_empties_from_list(data, strip_values_tuple=(None, str(), list(), dict(),)):
+def strip_empties_from_list(data, strip_values_tuple=(None, str(), [], {},)):
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/plugins/module_utils/strip_empties.py]: maybe we should also replace `str()` with `""` to be consistent.

**Comment by carlbuchmann** [comment]
ANSIBLE_METADATA has been deprecated, so I removed this code block: https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#ansible-metadata-block

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**